client/src/modules/Schedule/components/TableView/TableView.test.tsx

```diff
@@ -1,52 +1,25 @@
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen, within } from '@testing-library/react';
 import TableView, { TableViewProps } from './TableView';
 import * as ReactUse from 'react-use';
-import { ALL_TAB_KEY, ColumnKey } from 'modules/Schedule/constants';
-import { CourseScheduleItemDto, CourseScheduleItemDtoStatusEnum } from 'api';
+import { ALL_TAB_KEY, ColumnKey, ColumnName } from 'modules/Schedule/constants';
+import { CourseScheduleItemDto, CourseScheduleItemDtoStatusEnum, CourseScheduleItemDtoTagEnum } from 'api';
+import { ScheduleSettings } from 'modules/Schedule/hooks/useScheduleSettings';
 
 const StatusEnum = CourseScheduleItemDtoStatusEnum;
-
-const PROPS_MOCK: TableViewProps = {
-  settings: {
-    timezone: 'Europe/Moscow',
-    setTimezone: jest.fn,
-    tagColors: {
-      coding: '#722ed1',
-      'cross-check': '#13c2c2',
-    },
-    setTagColors: jest.fn,
-    columnsHidden: [],
-    setColumnsHidden: jest.fn,
-    tagsHidden: [],
-    setTagsHidden: jest.fn,
+const TagsEnum = CourseScheduleItemDtoTagEnum;
```

Comment:
By the way, you can just rename it during the import (not sure that approach suggested by me is better than yours but it may be shorten sometimes :D

Reply:
Good point, thank you. Fixed